src/provider/model/FinEnvironment.ts

```diff
@@ -60,25 +64,31 @@ export class FinEnvironment extends AsyncInit implements Environment {
         }
     }
 
-    public wrapApplication(appInfo: Application, identity: Identity, channel: ContextChannel): AppWindow {
+    public wrapApplication(appInfo: Application, identity: Identity, entityType: EntityType, channel: ContextChannel): AppConnection {
```

Comment:
My PR renames this to `wrapWindow`. Should this be `wrapConnection`?

Reply:
Yeah, `wrapConnection` IMO. It can remain as `wrapWindow` if it specifically only ever deals with windows, but this method can be used for any type of connection.

Reply:
Think do in this PR